Environment/electricitymaps.5m.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+
+# <xbar.title>Electricity Maps</xbar.title>
+# <xbar.version>v1.0</xbar.version>
+# <xbar.author>Olivier Corradi</xbar.author>
+# <xbar.author.github>corradio</xbar.author.github>
+# <xbar.desc>This plugin displays how clean the electricity is in your current location</xbar.desc>
+# <xbar.dependencies>python, Electricity Maps API</xbar.dependencies>
+# <xbar.abouturl>https://docs.electricitymaps.com/</xbar.abouturl>
+# <xbar.image>https://github.com/user-attachments/assets/dd55b506-a416-4f7d-b183-af31088f76a7</xbar.image>
```

Comment:
Could you make this a direct link to the image? This link redirects to an AWS link.